compiler/rustc_parse/src/errors.rs

```diff
@@ -2645,3 +2645,18 @@ pub(crate) struct MalformedCfgAttr {
     pub span: Span,
     pub sugg: &'static str,
 }
+
+#[derive(Diagnostic)]
+#[diag(parse_unknown_builtin_construct)]
+pub(crate) struct UnknownBuiltinConstruct {
+    #[primary_span]
+    pub span: Span,
+    pub ident: Symbol,
```

Comment:
```suggestion
    pub name: Symbol,
```
Nit: conventionally something is named `ident` only if it's actually an `Ident`.